doc/15-troubleshooting.md

```diff
@@ -230,6 +232,97 @@ icinga2 7764 nagios  mem    REG                8,5   744776 2631636 /usr/lib/x86
 #
 ```
 
+### Optimise apply rules and group assign conditions
+
+#### Remove actually unused apply rules
+
+If `icinga2 daemon -C` warns you like shown below and "it's okay" for you
+as everything behaves as desired, consider removing
+or commenting out the respective apply rule.
```

Comment:
The first part of the sentence sounds unclear to me. I think you want to say something like "Check if there is an error in your configuration. If the rule should indeed create no objects, consider ..."

Reply:
Well, tech-ly it’s not an error, just a waste of reload time. And given `icinga2 daemon -C` the user doesn’t have to lookup that command by themselves.

Reply:
An error as in user error. Something like misspelling something in a condition that should produce a match but doesn't